### DISCUSSION TITLE
群辉的docker用哪个镜像才能下到新一点的青龙呢？

### DISCUSSION THREAD

**Discussion by ema1995cylove** [whyour/qinglong#2123] [Q&A]
如题，用了阿里云的镜像只能下到2.10的版本，属实差太多了
求助！！

**Comment by whyour** [comment]
> 如题，用了阿里云的镜像只能下到2.10的版本，属实差太多了
> 
> 求助！！

阿里的镜像源不保证latest是最新的版本，要么拉取指定版本，要么用官方镜像源
